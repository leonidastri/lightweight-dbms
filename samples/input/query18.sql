select R.G, B.D, S.A from Sailors S, Boats B, Reserves R where S.A = B.E AND B.D > R.H AND S.A = R.G;