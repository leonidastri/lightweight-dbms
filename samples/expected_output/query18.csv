1,103,1
1,103,1
2,107,2